index.html

```diff
@@ -2094,12 +2197,34 @@ <h2>
           A boolean that indicates whether the <a>user agent</a> SHOULD collect
           and return the billing address associated with a <a>payment
           method</a> (e.g., the billing address associated with a credit card).
-          Typically, the user agent will return the billing address as part of
-          the {{PaymentMethodChangeEvent}}'s <a>methodDetails</a>. A merchant
-          can use this information to, for example, calculate tax in certain
-          jurisdictions and update the displayed total. See below for privacy
-          considerations regarding <a href="#user-info">exposing user
-          information</a>.
+          The <a>user agent</a> returns the billing address as part of the
+          {{PaymentMethodChangeEvent}}'s {{PaymentMethodChangeEvent/methodDetails}} attribute and the
```

Comment:
Is there a standard formatter that we use for W3C specs? I tried [prettier](https://prettier.io) on this file but it modifies a lot of existing content. So I was planning to land this patch, and then run `prettier` once to just adjust the format. I'd be happy to help document the steps to keep a spec style-compliant.

Reply:
Yes, sorry. I thought it was listed in one of the guides. We use:

https://github.com/htacg/tidy-html5

If you are on a Mac:
```
brew install tidy-html5
```

And to run it:

```
tidy -config tidyconfig.txt -o index.html index.html
```

Reply:
Thanks! I tried the command, but `tidy` complains about not recognizing &lt;section&gt;. Do you know why?
```
line 94 column 5 - Error: <section> is not recognized!
line 94 column 5 - Warning: discarding unexpected <section>
[omitted similar lines]
645 warnings, 148 errors were found! Not all warnings/errors were shown.

This document has errors that must be fixed before
using HTML Tidy to generate a tidied up version.
```

Reply:
oh, it must not be finding the right tidy version.... check:

```
tidy --version
```

Should say "HTML Tidy for Apple macOS version 5.6.0". Otherwise, it didn't install correctly. 

No worries if we can't tidy it right now tho... we can always do that last.